### DISCUSSION TITLE
Input Output Example not working on my board

### DISCUSSION THREAD

**Discussion by RuudErmers** [pschatzmann/arduino-audiokit#93] [Q&A] [answered]
Hi, thanks for this great library!
Unfortunately, the Input Output Example is not working on my Thinker Board.
It has an AC-101 and the output example is working.
When I connect my phone to the Input (Line In) and flash the Input-Output example, I don't hear anything.
What am I doing wrong? 
I am using PlatformIO.. (but get the same result from Arduino-IDE)

**Comment by pschatzmann** [comment]
I don't remember the correct setting for the AC101 to use  microphones. 
Did you try AUDIO_HAL_ADC_INPUT_LINE2 ?

In addition I recommend to use the [AudioTools library](https://github.com/pschatzmann/arduino-audio-tools) with the [corresponding examples](https://github.com/pschatzmann/arduino-audio-tools/tree/main/examples/examples-audiokit)

**Reply by RuudErmers** [reply]
OK, using Auditools now. In the Audiotools a copier is used and it uses AUDIO_HAL_ADC_INPUT_LINE2 . But, alas, changing to AUDIO_HAL_ADC_INPUT_LINE2 didn't help. I really think I am doing some stupid here..., "all" other 'output' examples just work. If I replace the line In/Out through the board by a 3.5 female female adapter I hear music :} .
I never understood the dip switches, could they be a cause?

**Comment by pschatzmann** [comment, marked as answer]
No he dip switches only impact the SD card.
If AUDIO_HAL_ADC_INPUT_LINE1 works with the Aux In, then the AUDIO_HAL_ADC_INPUT_LINE2 should work with the micrphones.

Maybe it is just an issue with the volume or the microphones are broken...

**Reply by RuudErmers** [reply]
Note that I allready had my phone connected to Line In (not microphone). I took another board, it didn't work at first, tried and switched and finally, suddenly, all my three boards just worked! (I use AUDIO_HAL_ADC_INPUT_LINE2). I think it had to do wit unplugging the board. anyway it works! Great thanks
